src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -224,6 +230,34 @@ def write_data(
             log.error("No uds config found")
             return
 
+    def sender_run(self, period: int, stop_event: threading.Event) -> None:
```

Comment:
Should be marked as private, I don't think a user should call the method that will just result in an endlessly blocking call

Reply:
Done